src/eos-companion-app-integration-helper.c

```diff
@@ -608,26 +658,61 @@ eos_companion_app_service_finish_load_application_icon_data_async (GAsyncResult
   return g_task_propagate_pointer (G_TASK (result), error);
 }
 
+static EosCompanionAppServiceAppInfo *
+load_application_info (const gchar  *app_id,
+                       GError      **error)
+{
+  g_autoptr(GDesktopAppInfo) app_info = NULL;
+  g_autofree gchar *runtime_name = NULL;
+  g_autofree gchar *runtime_version = NULL;
+  const gchar *eknservices_name = NULL;
+  const gchar *search_provider_name = NULL;
+  g_autofree gchar *runtime_spec = NULL;
+
+  runtime_spec = eos_companion_app_service_get_runtime_name_for_app_id (app_id, error);
```

Comment:
It's confusing that this function says that it returns a `runtime_name`, yet you call the variable `runtime_spec` and you need to call `parse_runtime_spec()` to actually get something called `runtime_name`

Reply:
Yeah, I'll rename it